sopel/formatting.py

```diff
@@ -60,6 +62,67 @@
 CONTROL_REVERSE = '\x16'
 """The control code to start or end reverse-color formatting."""
 
+CONTROL_NON_PRINTING = [
+    '\x00',
+    '\x01',
+    # '\x02', CONTROL_BOLD
+    # '\x03', CONTROL_COLOR
+    # '\x04', CONTROL_HEX_COLOR
+    '\x05',
+    '\x06',
+    '\x07',
+    '\x08',
+    '\x09',
+    '\x0a',
+    '\x0b',
+    '\x0c',
+    '\x0d',
+    '\x0e',
+    # '\x0f', CONTROL_NORMAL
+    '\x10',
+    # '\x11', CONTROL_MONOSPACE
+    '\x12',
+    '\x13',
+    '\x14',
+    '\x15',
+    # '\x16', CONTROL_REVERSE
+    '\x17',
+    '\x18',
+    '\x19',
+    '\x1a',
+    '\x1b',
+    '\x1c',
+    # '\x1d', CONTROL_ITALIC
+    # '\x1e', CONTROL_STRIKETHROUGH
+    # '\x1f', CONTROL_UNDERLINE
```

Comment:
Whatever the final implementation, it (IMO) should include character code `7F` in what gets stripped.

```suggestion
    # '\x1f', CONTROL_UNDERLINE
    '\x7f',
```

Reply:
Will do!